kinesis-video-gst-demo/kinesis_video_gstreamer_audio_video_sample_app.cpp

```diff
@@ -630,11 +645,11 @@ void kinesis_video_init(CustomData *data) {
 void kinesis_video_stream_init(CustomData *data) {
 
     STREAMING_TYPE streaming_type = DEFAULT_STREAMING_TYPE;
-    bool use_absolute_fragment_times = DEFAULT_ABSOLUTE_FRAGMENT_TIMES;
+    data->use_absolute_fragment_times = DEFAULT_ABSOLUTE_FRAGMENT_TIMES;
```

Comment:
we dont really need this use_absolute_fragment_times variable. Previously we are using relative mode for realtime and absolute mode for offline. Now they are both absolute.

Reply:
I think we still want it as offline will always go absolute while live can be modify to support relative mode by changing the DEFAULT_ABSOLUTE_FRAGMENT_TIMES to FALSE without affecting offline mode if relative time mode is needed for some reason.